metarecord/views/export.py

```diff
@@ -11,6 +14,34 @@
 logger = logging.getLogger(__name__)
 
 
+def create_saved_jhs_xml():
+    try:
+        exporter = JHSExporter(output=True)
+        xml = exporter.create_xml()
+        save_jhs_export_to_file(xml)
+
+        return xml
+
+    except JHSExporterException as e:
+        logger.error('Exception while creating a cached JHS191 XML export: %s' % e)
+        raise APIException('Could not create an XML export.')
+
+
+def save_jhs_export_to_file(xml):
+
+    directory = os.path.join(settings.MEDIA_ROOT, "export")
```

Comment:
I wouldn't save the file under media. The xml file isn't media file in the same way user uploaded images and other files are. I guess it doesn't really matter in this case to be honest but I think it's still worth mentioning.

Reply:
Maybe @vikoivun can comment this better.

Reply:
@vikoivun developers work for this sprint ended Friday, so I would like to get this feature ready, so can some of our own developers help - review this pr and help with #216

Reply:
Probably not worth creating a separate directory for just this file. We'll fix things later if this causes problems.